<launch>

  <!-- Launch Rviz -->
  <node pkg="rviz" type="rviz" name="rviz">
  </node>

  <group ns="robot" >

    <!-- Launch the simulator -->
    <node pkg="mobile_robots" type="simulator" name="simulator" >
    </node>

    <!-- Launch the controller -->
    <node pkg="mobile_robots" type="staticfeedback_2_0" name="staticfeedback_2_0" >
    </node>

    <!-- Launch the target -->
    <node pkg="mobile_robots" type="circular_target" name="circular_target" >
    </node>

    <!-- Launch the robot -->
    <node pkg="mobile_robots" type="robot_2_0" name="robot_2_0" >
    </node>

  </group>

  <group ns="follower" >

    <!-- Launch the simulator -->
    <node pkg="mobile_robots" type="simulator" name="simulator" >
    </node>

    <!-- Launch the controller -->
    <node pkg="mobile_robots" type="staticfeedback_2_0" name="staticfeedback_2_0" >
      <remap from="/follower/TargetReference" to="/robot/RobotPose" />
    </node>

    <!-- Launch the robot -->
    <node pkg="mobile_robots" type="robot_2_0" name="robot_2_0" >
    </node>



  </group>



</launch>
